.github/pull_request_template.md

```diff
@@ -1,5 +1,11 @@
+**Describe your changes**
+Briefly explain what you did and provide context for a clearer understanding.
+
+**How to Reproduce**
+Provide steps to recreate and test the changes.
+
 **Tasks**
-- [ ] PR name contains story or task reference
+- [ ] Short and clear PR title with story or task reference
```

Comment:
I sometimes use the story reference for searching for all PR's relating to that story and I personally think also needing to check all the task references would be annoying but maybe it makes sense for larger stories?